### PR TITLE
aws bucket name & bug fix

### PR DESCRIPTION
**Pull request by knockknockyoo**
1. submission bucket name changed
2. organization scheme changed
3. organization id already stored in the session

### CODE REVIEW

File: services/submission.js
Comment (vshand11) [inline services/submission.js:1]: what's the reason for the following change? is userInfo?.organization?.orgID always works?

Comment (knockknockyoo) [reply services/submission.js:1]: I've rolled back the change.

Comment (vshand11) [reply services/submission.js:1]: Approved

**Review by vshand11 — APPROVED** [review]
approved

**Pull request merged**